content/drafts/normal-operation-pre-taproot.md

```diff
@@ -0,0 +1,624 @@
+sides will need to re-transmit any `update_*` messages that were not yet
+committed on the other side’s commitment transaction.
+
+# Closing a channel cooperatively
+
+Alice and Bob sure had some good times together but all good things must come to
+an end. Closing a channel in a cooperative way requires the two peers to decide
+on a final closing transaction that will spend from the funding transaction and
+will pay each of them their final channel balance immediately.
+
+### :gear: Step 17: Bob -> Alice: `shutdown`
+
+Bob has decided that it is time to cut ties and sends Alice the `shutdown`
+message.
+
+![](/normalChanOp/shutdown.png#center)
+
+The `shutdown` message contains the `scriptpubkey` that Bob would like his final
+channel balance to be sent to in the closing transaction. Once Bob has sent this
+message, he may no longer send any new `update_add_htlc` messages. He may only
+send HTLC removal and `update_fee` messages. When Alice receives this message
+from Bob, she must respond with her own `shutdown` message and may also no 
+longer send any new `upate_add_htlc` messages. Alice and Bob now need to wait 
+until all remaining HTLCs have been cleared from both commitment transactions. 
+Since the closing transaction will spend from the funding transaction and 
+explicitly looks different from the commitment transactions, I’ll re-introduce 
+some of the details in to the state diagram:
+
+![](/normalChanOp/1-initial.png#center)
+
+Once all the HTLCs have been cleared, which in our example is already the case,
+they can start negotiating a fee to use for the final closing transaction. The
+funder of the channel must start this negotiation. Let’s assume that the funder
+of this channel was Alice.
+
+### :gear: Step 18: Alice -> Bob: `closing_signed`
+
+Alice will first choose a fee-rate that she thinks is appropriate for the
+closing transaction. She will then use that fee-rate to complete the
+construction of the closing transaction and will sign it. She then sends the
+`closing_signed` message to Bob:
+
+![](/normalChanOp/closing_signed.png#center)
+
+The `fee_satoshis` field tells Bob the fee-rate that Alice used to construct the
+first closing transaction proposal and the `signature` contains Alice’s
+signature for this proposal. She may optionally also include
+the `min_fee_satoshis` and `max_fee_satoshis` fields in order to let Bob know
+that if he disagrees with her proposed `fee_satoshis`, then he may send a
+counterproposal as long as his counterproposal lies between the provided
+minimum and maximum values.
+
+At this point, the channel state looks as follows:
+
+![](/normalChanOp/32-closing-alice-prop.png#center)
+
+There are two valid commitment transactions that can be signed at any time by
+each party to perform a force close, and there is one closing transaction
+proposal that uses a fee-rate of `x` sats-per-byte. This closing transaction
+currently only has Alice’s signature and so is not yet valid.
```

Comment:
May be worth mentioning that Bob could go ahead and broadcast the coop close at this point and not send back a `closing_signed`

Reply:
added!